app/views/layouts/application.html.erb

```diff
@@ -21,7 +21,11 @@
     <% end %>
     <main role="main">
       <div class="container">
-        <%= render 'layouts/messages' %>
+        <div id="flash">
```

Comment:
Not exactly sure if we need this, I remembered following this https://stackoverflow.com/questions/23967390/rails-flash-notice-via-ajax/23967474

Then forgetting why I did it in the first place.